pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -53,6 +64,26 @@ type ClusterAutoscalerSpec struct {
 	// This option has priority over log level set by the `CLUSTER_AUTOSCALER_VERBOSITY` environment variable.
 	// +kubebuilder:validation:Minimum=0
 	LogVerbosity *int32 `json:"logVerbosity,omitempty"`
+
+	// Sets the type and order of expanders to be used during scale out operations.
+	// This option specifies an ordered list, highest priority first, of expanders that
+	// will be used by the cluster autoscaler to select node groups for expansion
+	// when scaling out.
+	// Expanders instruct the autoscaler on how to choose node groups when scaling out
+	// the cluster. They can be specified in order so that the autoscaler will utilize
+	// the result from one expander on the next, for example if the first expander option
```

Comment:
```suggestion
	// the result from one expander or the next, for example if the first expander option
```

Reply:
that is actually correct, this is meant to tell people that the expanders work like filters.

if the user specifies `expanders: [LeastWaste, Random]`, the autoscaler will first determine which node group(s) will satisfy least waste, if there are multiple groups selected, then the random expander will choose one of those.

Reply:
maybe it would be clearer if i said this?

> They can be specified in order so that the result from the first expander is used as input to the second, and so forth.

Reply:
Yeah I think that would be a clearer way to word it